### PR TITLE
[oneDPL][doc]  limitation for permutation_iterator using

### PR DESCRIPTION
**Pull request by MikeDvorskiy**
[oneDPL][doc] + limitation for permutation_iterator using

### CODE REVIEW

File: documentation/library_guide/parallel_api/iterators.rst [hunk 1]
```diff
@@ -92,7 +92,8 @@ header.  All iterators are implemented in the ``oneapi::dpl`` namespace.
   a functor whose index operator defines the mapping from the ``permutation_iterator`` index to the index of the
   source iterator. The ``permutation_iterator`` is useful in implementing applications where noncontiguous
   elements of data represented by an iterator need to be processed by an algorithm as though they were contiguous.
-  An example is copying every other element to an output iterator.
+  An example is copying every other element to an output iterator. The source iterator cannot be a host-side iterator
```
Comment (dcbenito) [inline documentation/library_guide/parallel_api/iterators.rst:95]: My suggestion:

An example is copying every other element to an output iterator. The source iterator cannot be a host-side iterator when algorithms are executed with device policies.

**Review by dcbenito — CHANGES_REQUESTED** [review]

File: documentation/library_guide/parallel_api/iterators.rst (hunk 1 above)
Comment (MikeDvorskiy) [reply documentation/library_guide/parallel_api/iterators.rst:95]: resolved.
@dcbenito  could you please re-review the changes?

Comment (dcbenito) [reply documentation/library_guide/parallel_api/iterators.rst:95]: My suggestion: The source iterator cannot be a host-side iterator in cases where algorithms are executed with device policies.

**Comment by MikeDvorskiy** [comment]
@dcbenito, do you have another objections to merge the changes?

**Comment by dcbenito** [comment]
LGTM, changes approved.

**Review by dcbenito — APPROVED** [review]

**Pull request merged**